docs/user-guide/timepix_images.ipynb

```diff
@@ -0,0 +1,425 @@
+    "import sciline as sl\n",
+    "\n",
+    "from typing import NewType, TypeVar\n",
+    "\n",
+    "Sample = NewType('Sample', int)\n",
+    "OpenBeam = NewType('OpenBeam', int)\n",
+    "RunType = TypeVar('RunType', Sample, OpenBeam)\n",
+    "\n",
+    "\n",
+    "class FilePath(sl.Scope[RunType, pathlib.Path], pathlib.Path): ...\n",
+    "\n",
+    "\n",
+    "class TiffImage(sl.Scope[RunType, sc.DataArray], sc.DataArray): ...\n",
+    "\n",
+    "\n",
+    "NormalizedImage = NewType('NormalizedImage', sc.DataArray)\n",
+    "\n",
+    "\n",
+    "class TofShift(sl.Scope[RunType, int], int): ...\n",
+    "\"\"\"Tof shifted index.\"\"\"\n",
+    "\n",
+    "\n",
+    "DEFAULT_TOF_SHIFT = TofShift(0)\n",
+    "\n",
+    "\n",
+    "def load_image(\n",
+    "    path: FilePath[RunType], tof_shift: TofShift[RunType] = DEFAULT_TOF_SHIFT\n",
+    ") -> TiffImage[RunType]:\n",
+    "    easier_dims = {'dim_0': 'x', 'dim_1': 'y', 'dim_2': 'tof'}\n",
+    "    img = load_tiff(path).rename_dims(easier_dims)\n",
+    "    # Hard-code the time coordinate\n",
+    "    tof_coord = sc.linspace(\n",
+    "        dim='tof', start=0.0, stop=40.0, num=img.sizes['tof'], unit='ms'\n",
+    "    )\n",
+    "    tof_coord = sc.concat([tof_coord[-tof_shift:], tof_coord[:-tof_shift]], dim='tof')\n",
+    "    img.coords['tof'] = tof_coord\n",
+    "    # Hard-code the positional coordinates\n",
+    "    img.coords['x'] = sc.linspace(\n",
+    "        dim='x', start=0, stop=10.0, num=img.sizes['x'] + 1, unit='cm'\n",
+    "    )\n",
+    "    img.coords['y'] = sc.linspace(\n",
+    "        dim='y', start=0, stop=10.0, num=img.sizes['y'] + 1, unit='cm'\n",
+    "    )\n",
+    "    sorted_img = sc.sort(img, 'tof')\n",
+    "    sorted_img.coords['tof'] = sc.linspace(\n",
+    "        dim='tof', start=0.0, stop=40.0, num=img.sizes['tof'] + 1, unit='ms'\n",
+    "    )\n",
+    "    return sorted_img\n",
+    "\n",
+    "\n",
+    "ReplaceNanToZero = NewType('ReplaceNanToZero', bool)\n",
+    "DEFAULT_REPLACE_NAN_TO_ZERO = ReplaceNanToZero(True)\n",
+    "LTotal = NewType('LTotal', sc.Variable)\n",
+    "\n",
+    "\n",
+    "class WavelengthPixelCounts(sl.Scope[RunType, sc.DataArray], sc.DataArray): ...\n",
+    "\n",
+    "\n",
+    "def timepix_tof_to_wavelength(\n",
+    "    l_total: LTotal, tiff_image: TiffImage[RunType]\n",
+    ") -> WavelengthPixelCounts[RunType]:\n",
+    "    from scippneutron.conversion import graph\n",
+    "\n",
+    "    tiff_image.coords['Ltotal'] = l_total\n",
+    "    plane_graph = graph.tof.kinematic(\"tof\")\n",
+    "    transformed = tiff_image.transform_coords('wavelength', graph=plane_graph)\n",
+    "    return WavelengthPixelCounts[RunType](transformed)\n",
+    "\n",
+    "\n",
+    "def normalize_image(\n",
+    "    sample_img: WavelengthPixelCounts[Sample],\n",
+    "    openbeam_img: WavelengthPixelCounts[OpenBeam],\n",
+    "    replace_nan_to_zero: ReplaceNanToZero = DEFAULT_REPLACE_NAN_TO_ZERO,\n",
```

Comment:
Sciline doesn't see the default values anyways but I'll remove `DEFAULT_REPLACE_NAN_TO_ZERO` one since it's not expected to be used out side of sciline graph.
But I would like to keep the `DEFAULT_TOF_SHIFT` sciline that one will be used as it is, as a helper for loading other tiff stacks.